code/datums/objectives/objective.dm

```diff
@@ -0,0 +1,16 @@
+/datum/objective
+	var/datum/mind/owner = null //Is the objective just yours?
+	var/datum/faction/faction = null // Is the objective faction-wide?
+	var/explanation_text = "Just be yourself." //What that person is supposed to do.
+	var/is_void = 0 // Universe is doomed what's the point.
+
+/datum/objective/New(var/text)
+	if(text)
+		explanation_text = text
+
+/datum/objective/proc/IsFulfilled()
+	if(is_void)
+		return FALSE
+
+/datum/objective_holder
+	var/list/datum/objective/held = list()
```

Comment:
In faction datums, I assumed this variable would be called objectives, should I change faction datums or are you good to change this variable?

Reply:
I can change it no problem